pom.xml

```diff
@@ -359,6 +360,11 @@ under the License.
       <version>${jettyVersion}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>${slf4j.version}</version>
```

Comment:
Is it used for something else than tests? If so - could the scope be reduced to runtime?

Reply:
oh good catch! fixed with cebfb72